bitnami/mongodb/templates/replicaset/scripts-configmap.yaml

```diff
@@ -87,6 +89,7 @@ data:
     {{- end }}
 
     echo "Advertised Hostname: $MONGODB_ADVERTISED_HOSTNAME"
+    echo "Advertised Port: $MONGODB_ADVERTISED_PORT_NUMBER"
```

Comment:
This comment should also be added in line 145.

Reply:
Thanks,
Added in line 145